metals/src/main/scala/scala/meta/internal/metals/MetalsServerConfig.scala

```diff
@@ -66,6 +66,10 @@ final case class MetalsServerConfig(
       "bloop.embedded.version",
       BuildInfo.bloopVersion
     ),
+    askToReconnect: Boolean = MetalsServerConfig.binaryOption(
```

Comment:
Instead of introducing this as a server property, can we just introduce it in `InitializationOptions`?

Reply:
Client editor should never set this. It's only to be set by a user that knows what they are doing. Usually via `.jvmopts` file in the workspace. The default is fine for all clients.

Reply:
ahhh ok, that makes sense. 👍